src/traces/scatter/attributes.js

```diff
@@ -166,6 +167,10 @@ module.exports = {
             'these elements will be seen in the hover labels.'
         ].join(' ')
     },
+
+    texttemplate: texttemplateAttrs({arrayOk: true}, {
```

Comment:
I think you can :hocho: that `{arrayOk: true}` part. Doesn't `texttempalteAttrs` return an object with `arrayOk: true` by default?

Reply:
Done in cbbcfcbe3635eb34a51ef2e8a8b35f0c885c7323